src/topp/CometAdapter.cpp

```diff
@@ -282,8 +291,10 @@ class TOPPCometAdapter :
     isotope_error["0/1"] = 1;
     isotope_error["0/1/2"] = 2;
     isotope_error["0/1/2/3"] = 3;
-    isotope_error["-8/-4/0/4/8"] = 4;
-    isotope_error["-1/0/1/2/3"] = 5;
+    isotope_error["-1/0/1/2/3"] = 4;
+    isotope_error["-1/0/1"] = 5;
+    isotope_error["-3/-2/-1/0/1/2/3"] = 6;
+    isotope_error["-8/-4/0/4/8"] = 7;
```

Comment:
these two cannot be set by the user. Please add them to the list of possible values.

Reply:
Added. 
I used to wonder why they shouldn't be added to the list of valid values. Perhaps the Comet's documentation needs to be updated.
From the comet documentation :
> Valid values are 0, 1, 2, 3, 4 and 5:
>+  0 analyzes no isotope offsets, just the given precursor mass
>+ 1 searches 0, +1 isotope offsets
>+ 2 searches 0, +1, +2 isotope offsets
>+ 3 searches 0, +1, +2, +3 isotope offsets
>+ 4 searches -1, 0, +1, +2, +3 isotope offsets
>+ 5 searches -1, 0, +1 isotope offsets
>+ 6 searches -3, -2, -1, 0, +1, +2, +3 isotope offsets
>+ 7 searches -8, -4, 0, +4, +8 isotope offsets (for +4/+8 stable isotope labeling)

Reply:
yes, I wrote the Comet maintainer and asked to fix this.